docs/charter.md

```diff
@@ -10,7 +14,43 @@ ICU4X will be built from the start with several key design constraints:
 3. Availability and ease of use in multiple programming languages.
 4. Written by i18n experts to encourage best practices.
 
-ICU4X will provide an API surface in the target client-side platforms, including the web platform, iOS, Android, WearOS, WatchOS, Flutter, and Fuchsia, supported in programming languages including Rust, JavaScript, Objective-C, Java, Dart, and C++.
+# Scope
+
+ICU4X aims to provide a set of APIs sufficient to internationalize basic modern software.
+
+With the focus on client-side, it will initially focus on the scope covered by the ECMA-402 extended to cover needs of more low level systems such as Gecko or Fuchsia.
+
+The current target is to cover all of ECMA-402 needs, and a modern subset of ICU4X APIs needed for software platforms such as Fuschia and Gecko.
+
+One of the benchmarks on decisions related to whether a given API belongs to ICU4X will be if it requires Unicode/CLDR data and would benefit from being managed together with other APIs that rely on the same version of Unicode and CLDR data.
```

Comment:
This is the most important paragraph in my opinion.  We should list out what all of the benchmarks are.  Suggestions:

- Useful to a broad audience of client-side applications
- Depends on Unicode or CLDR data or algorithms
- Benefits from being versioned alongside Unicode and CLDR data

Reply:
i have a suggested change below 👇